### ISSUE TITLE
bge.render.makeScreenshot("filename") saves all white images, still in OSX 10.13.6 

### ISSUE THREAD

**Issue by sevi78** [UPBGE/upbge#1801]
i have found this post: 
https://github.com/UPBGE/upbge/issues/1691

i still have the same proplem. Screenshot is not working as it should, only crashed white images with some noise inside ..

**Comment by youle31** [comment]
I can't test on mac (I've no mac).

I just tested with this file on windows (22h2) and linux (mint 21.something) with my machine (RTX 3070):

[MakeSceenShotTest.zip](https://github.com/UPBGE/upbge/files/10985985/MakeSceenShotTest.zip)

And It seemed to work.

A previous issue on linux  related to MakeScreenShot was fixed here: https://github.com/UPBGE/upbge/commit/ee50a07006027365ff3da3806a29448f2d73723e

Could you provide more infos on your OS, Hardware (GPU/CPU...), is it using metal backend, exact steps to reproduce the issue, upbge versions tested (you need to test last version based on blender 3.6 alpha) please? Thanks. (https://github.com/UPBGE/upbge/wiki/Report-a-bug-guide)

**Comment by sevi78** [comment]
yes, i have made a simple test file here: https://github.com/sevi78/UPBGE-Tests/tree/main/VideoTextureTest
i am using:

UPBGE 0.30.0 
hash2df03652494
Branch:upbge-v0.3-release
from https://upbge.org/#/

on: 

MacBook Pro (Retina, 15-inch, Mid 2014)
2.2 GHz Intel Core i7
16 GB 1600 MHz DDR3
Macintosh HD
Intel Iris Pro 1536 MB

I am using 2 Monitors, but have testet also without second monitor.

There is nothign to see in console, any bugs or problems from upbge: 

```
Blender Game Engine Started
Blender Game Engine Finished
Debug: 1660, 992
rcti: : xmin 0, xmax 1659, ymin 613, ymax 1604 (1659x991)

Blender Game Engine Started
Blender Game Engine Finished



```
i am pretty shure, it has something to do with my old system 


Is there any repo to download the latest 3.6 Version---- i am not shure to manage to build it my own...
I testet now your file aswell, only black bar at the bottom. 
I will make a test now on windows and give you answer if it runs.

thanks.

**Comment by youle31** [comment]
https://mega.nz/folder/t9EEFSaS#JPiOPSInCZyU-SW_-rhEOQ

You can download development versions from here (there is a link on upbge website)

I bet the bug you mention is fixed with the commit I mentionned... In this case you can try to find a development version which works for you and use it... (Generally, builds which are just a bit before blender release dates can work better)

**Comment by sevi78** [comment]
i have testet now on osx: could not install any of the builds because my system is too old. BUT:
i testet aswell on Windows 10, where UPBGE 0.3.0 didnt work, but UPBGE 0.36.0 works fine. 
So i would say, thats fine.
Thanks for your time
